h-w1

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Do not copy code from the console to file. You should write your code directly in the file and then push it to the repository.